test/scripts/params_and_secrets.txtar

```diff
@@ -0,0 +1,17 @@
+# [execute] attest fn call
+stdin fn-call.json
+exec bky-as attest-fn-call
+cp stdout result.json
+
+# [check] assert stdout contains correct decrypted information
+exec jq -c -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result.json
+stdout '{"Success":true,"Error":"","Value":{"decrypted_data":"your decrypted information"}}'
+
+# [execute] attest fn call that errors
+stdin fn-call-error.json
+exec bky-as attest-fn-call
+cp stdout result-error.json
+
+# [check] assert stdout does not contain decrypted information
+exec jq -c -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result-error.json
+stdout '{"Success":false,"Error":"decrypting data: incorrect password","Value":null}'
```

Comment:
One thing that I think is ok for now but could become a thing in the future: My understanding is that golang serializes json in the same order as the struct, but this is an implementation detail and should not necessarily be relied on. For example, this discussion [here](https://github.com/golang/go/issues/6244#issuecomment-66084578). In using string matching rather than inspection with jq, we rely on the order. 

Again, I don't think this is something that needs to change, these tests are pretty straightforward, but it might be something to keep in mind and might be something we end up changing later.

Reply:
Same thing goes for the output json files. Seems worth doing the way you've suggested for now though, much easier in getting this up and running and may never end up being something we need to be concerned with.